happybase/table.py

```diff
@@ -558,6 +559,25 @@ def counter_inc(self, row, column, value=1):
         return self.connection.client.atomicIncrement(
             self.name, row, column, value)
 
+    def counter_incs(self, row, data):
+        """
+
+        This method increments of decrements the counter columns in the row
+        specified by `row`. The `data` argument is iteratble data type that
+        contains tuple of column and value, e.g. [(`cf:col`, 1), (`cf:col2`, 2)].
+
+        Note that unlike `counter_inc`, does not return value after incrementing.
+
+        :param str row: the row key
+        :param list data: list of tuple for columns and values
+        """
+        if data is not None and not isinstance(data, (list, tuple, izip)):
+            raise TypeError("'data' must be a iterable data types of tuple")
```

Comment:
About type of ```data```? I think that iterable collections are more general and efficiency than ```dict```, especially for bunch of data processing, and also ```dict``` can be used with ```iteritems``` method. But it seems a little bit verbose...